PowerShellGet/PSModule.psm1

```diff
@@ -788,7 +790,7 @@ function Publish-Module
 
         [Parameter(ParameterSetName="ModuleNameParameterSet")]
         [ValidateNotNullOrEmpty()]
-        [Version]
+        [string]

```

Comment:
You are right, this change is not required, tests passed with it reverted, so I will revert this.